src/langgraph-platform/use-remote-graph.mdx

```diff
@@ -1,289 +1,300 @@
 ---
-title: How to interact with the deployment using RemoteGraph
-sidebarTitle: Interact with the deployment using RemoteGraph
+title: How to interact with a deployment using RemoteGraph
+sidebarTitle: Interact with a deployment using RemoteGraph
 ---
-<Info>
-**Prerequisites**
-* [LangGraph Platform](/langgraph-platform/index)
-* [LangGraph Server](/langgraph-platform/langgraph-server)
-</Info>
 
-`RemoteGraph` is an interface that allows you to interact with your LangGraph Platform deployment as if it were a regular, locally-defined LangGraph graph (e.g. a `CompiledGraph`). This guide shows you how you can initialize a `RemoteGraph` and interact with it.
+[`RemoteGraph`](https://langchain-ai.github.io/langgraph/reference/remote_graph) is a client-side interface that allows you to interact with your [deployment](/langgraph-platform/deployment-options) as if it were a local graph. It provides API parity with [`CompiledGraph`](/oss/python/langgraph/graph-api#compiling-your-graph), which means that you can use the same methods (`invoke`, `stream`, `get_state`, etc.) in your development and production environments. This page describes how to initialize a `RemoteGraph` and interact with it.
 
-## Initializing the graph
+`RemoteGraph` is useful for the following:
+
+- Separation of development and deployment: Build and test a graph locally with `CompiledGraph`, deploy it to LangGraph Platform, and then [use `RemoteGraph`](#initialize-the-graph) to call it in production while working with the same API interface.
+- Thread-level persistence: [Persist and fetch the state](#persist-state-at-the-thread-level) of a conversation across calls with a thread ID.
+- Subgraph embedding: Compose modular graphs for a multi-agent workflow by embedding a `RemoteGraph` as a [subgraph](#use-as-a-subgraph) within another graph.
+- Reusable workflows: Use deployed graphs as nodes or [tools](https://langchain-ai.github.io/langgraph/reference/remote_graph#meth-as-tool), so that you can reuse and expose complex logic.
```

Comment:
This URL for `tool` might be better: `https://langchain-ai.github.io/langgraph/reference/remote_graph/#langgraph.pregel.remote.RemoteGraph.as_tool`

This URL actually scrolled down the page to the `as_tool` method for me.